example/cluster-machine-template-simple.yaml

```diff
@@ -38,16 +38,14 @@ spec:
     net:
       name: cluster-api-net
       clusterName: cluster-api
-      ipRange: "10.0.0.0/24"
+      ipRange: "10.0.0.0/16"
     subnets:
       - name: cluster-api-subnet-kcp
-        ipSubnetRange: "10.0.0.32/28"
+        ipSubnetRange: "10.0.0.32/24"
```

Comment:
Please changes with the same default values in exemple.

(Here also those /24 are not valid)